components/api/db/cellery-hub-db.bal

```diff
@@ -184,3 +184,36 @@ public function searchOrganizations(string orgName, int offset, int resultLimit)
         return null;
     }
 }
+
+public function searchUserOrganizations(string userId, string apiUserId, string orgName, int offset, int resultLimit) 
+returns json | error {
+    log:printDebug(io:sprintf("Performing data retreival on REGISTRY_ORGANIZATION table for userId : %s, Org name : \'%s\': ",
+    userId, orgName));
+    table<record {}> resTotal = check connection->select(SEARCH_USER_ORGS_TOTAL_COUNT, gen:Count, orgName, userId);
```

Comment:
We can do this with 2 queries right ? Run the SEARCH_USER_ORGS_QUERY_IMAGE_COUNT here and check the total count. Depending on the count you can run the second query.